README.md

```diff
@@ -1,62 +1,21 @@
-Backend for [https://eahub.org](https://eahub.org)
+<h1 align="center">EA Hub</h1>
+<p align="center">
+    <a href="https://eahub.org/">EAHub.org</a> |
+    <a href="https://discord.gg/NH5PybDUj9">Discord</a> |
+    <a href="https://blog.eahub.org/">News</a> |
+    <a href="https://meta.eahub.org/c/feature-requests/">Feedback</a>
```

Comment:
The news I would link to the https://meta.eahub.org

And perhaps same for feedback?